GUI/MainInstall.cs

```diff
@@ -390,6 +390,19 @@ private void PostInstallMods(object sender, RunWorkerCompletedEventArgs e)
                 RetryCurrentActionButton.Visible = false;
                 UpdateChangesDialog(null, installWorker);
             }
+            else if(!result.Key && installCanceled){
```

Comment:
We don't need to check `result.Key` here, because we're already in the `else` block of `if (result.Key)`.

Also, can you put the curly brace on a new line? This file follows that convention currently.